dl.sh

```diff
@@ -14,6 +13,6 @@ VER="eoan"
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libjavascriptcoregtk-4.0-dev/download
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libsoup2.4-dev/download
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libwebkit2gtk-4.0-dev/download
-
+./gir-dl.sh https://packages.debian.org/experimental/amd64/libpango1.0-dev/download
```

Comment:
I'm not sure if this is correct 😳
@EPashkin wdyt?

Reply:
It need something like  `ftp.de.debian.org/debian` in end to select mirror.
But this also replaces other "good" Pango-1.0.gir with "bad" one and Gir don't work with it.
I don't know how to fix it in right way: too many changes.
So I still suggest you try use your fork as is and ask question on generation if needed.

Reply:
I see I'll close the PR for now and revert the change to this file then. Thanks!